python-lib/dku_utils/tools_version.py

```diff
@@ -1,18 +1,31 @@
-import json
+import json, re
 from dku_kube.kubectl_command import run_with_timeout, KubeCommandException
 
 def get_kubectl_version():
     cmd = ['kubectl', 'version', '--client', '-o', 'json']
     out, err = run_with_timeout(cmd)
     return json.loads(out)['clientVersion']
 
+def get_kubectl_version_int(kubectl_version):
+    # the kubectl version downloaded from Amazon website has a minor version finishing by '+'
+    # keeping only the first numeric sequence for the minor version
+    regex_minor_int = re.compile("^[^0-9]*([0-9]+)")
+    search_results_minor_int = re.search(regex_minor_int, kubectl_version['minor'])
+    if len(search_results_minor_int.groups()) < 1:
```

Comment:
you need to check `search_results_minor_int is not None` too (ie the case where there is no match at all)